manufacturing/blower/README.md

```diff
@@ -68,14 +73,26 @@ We may have parts to send you from our inventory.
 [10rs]:   https://export.rsdelivers.com/product/jst/vhr-2n/jst-vhr-female-connector-housing-396mm-pitch-2/8201172
 [11amzn]: https://www.amazon.com/gp/product/B00N51OJJ4
 [12amzn]: https://www.amazon.com/gp/product/B011JC76OA
+[13rw]:   assets/blower_adapter.stl
+[14rw]:   assets/blower_adapter_holding_plate.dxf
+[15rw]:   assets/blower_driver_mounting.dxf
+[16rw]:   assets/blower_holding_plate.dxf
+
+
 
 ## Tools
 
 | Item | Quantity | Manufacturer  | Part #              | Price (USD) | Sources         | Notes |
 | ---- |---------:| ------------- | ------------------- | -----------:|-----------------| ----- |
 | a1   |        1 | Preciva       | PR-3254 (PR102001)  |    39.99 | [Z][a1amzn]     | Crimping tool for HX and HV connectors |
+| a2   |        1 | RespiraWorks  | Blower Adapter Gasket      | $1 (material) | [B][a2rw]            | Acrylic sheet cut from data in link |
```

Comment:
Is it really cut from acrylic?

Reply:
And what material is the gasket?

Reply:
And are these really tools or parts?

Reply:
Yeah the gaskets are cut using acrylic sheet.